crates/forge-runner/src/forge_config.rs

```diff
@@ -90,3 +90,19 @@ impl<'a> RuntimeConfig<'a> {
         }
     }
 }
+
+#[derive(Debug, PartialEq)]
+pub struct SierraTestCodePathConfig {
+    pub package_name: String,
+    pub sierra_test_code_dir: Utf8PathBuf,
+}
```

Comment:
No clue what it represents

Reply:
The configuration for sierra code path that is relevant for the whole package

Reply:
Removed